### DISCUSSION TITLE
Automatic Refresh Provider & Fetch API

### DISCUSSION THREAD

**Discussion by hci-ikahera** [Budibase/budibase#9977] [Help] [answered]
Hi everyone,
I want to ask some questions.
1. Is it possible to make provider refresh automatically every defined time?
If it is possible, how can I do it?

2. Is it possible to fetch API which does not have the same domain as the Budibase App?
I tried to fetch an API inside an embed component. But I got this error.

<img width="984" alt="Screenshot 2023-03-10 at 15 42 01" src="https://user-images.githubusercontent.com/95213000/224267124-f6def1db-178e-4597-b5a6-efaf53e19f6d.png">

I appreciate any kind of help.
Thank you.

Regards,
Ika

**Comment by melohagan** [comment, marked as answer]
Hey @hci-ikahera 

1. Not currently, there is an open issue: https://github.com/Budibase/budibase/issues/3992 (I see you found it!)
2. Have you / can you use the REST Connector instead of making the call within an embed component? You could also have a look at [Custom components](https://docs.budibase.com/docs/custom-component)

**Reply by hci-ikahera** [reply]
Okay @melohagan, thank you for your suggestion. May I know when feature related to #3992 will be available? Is it anytime soon?
